apiary.apib

```diff
@@ -5,6 +5,148 @@ HOST: https://operationcode.org/
 
 API endpoints that Operation Code's Rails backend makes available to its React frontend.
 
+## Users | Create [/api/v1/users{?first_name,last_name,email,zip,password,mentor}]
+
++ Parameters
+
+  + first_name (string, required) - First Name of the user in the form of an string
+  + last_name (string, required) - Last Name of the user in the form of an string
+  + email (string, required) - Email of the User in the form of a string
+  + zip (string, required) - Zipcode of the User in the form of a string
+  + password (string, required) - Password of the User in the form on a string
+  + mentor (boolean, required) - Is User requesting mentor in the form of a boolean
+
+### Create new user record [POST]
+
++ Request (application/json)
+
+    + Body
+
+            {            
+            first_name: "Mike",
+            last_name: "Johnson",
+            email: "dreams@operationcode.org"  ,     
+            zip: "80020" ,        
+            password: "hunter2",    
+            mentor: "true",   
+            }
+
+
++ Response 201 (application/json)
+
+        {
+        token: Bearer Access-Token
+        }
+
++ Response 422 (application/json)
+
+        {
+        errors: "Some error message"
+        }
+
+## Users | Update [/api/v1/users{?education_level,mentor,scholarship_info,employment_status,company_name,company_role,volunteer,branch_of_service,interests}]
+
++ Parameters
+
+  + education_level (string, required) - Users highest level of education in the form of a string
```

Comment:
These are all optional (vs. required)

Reply:
So if they submit no [params](https://github.com/OperationCode/operationcode_backend/blob/50aaf886e8ddda7a6ea05bbac4ba496df94ebc96/app/controllers/api/v1/users_controller.rb#L24) it returns a 422 or 200?